cmd/root.go

```diff
@@ -123,6 +124,7 @@ func setRootOpts(cmd *cobra.Command, opts *internal.ArgType) {
 	cmd.Flags().StringVarP(&opts.Out, "out", "o", "", "output path or file name")
 	cmd.Flags().StringVar(&opts.Suffix, "suffix", defaultSuffix, "output file suffix")
 	cmd.Flags().BoolVar(&opts.SingleFile, "single-file", false, "toggle single file output")
+	cmd.Flags().BoolVar(&opts.FilenameWithUnderscores, "with-underscores", false, "toggle underscores in file names")
```

Comment:
To align with the other options such as the `suffix` flag, how about removing `with` and naming it `underscore`?

Reply:
fixed [2c06c11](https://github.com/cloudspannerecosystem/yo/pull/141/commits/2c06c119297a1da65c2f5bbf9cb730048df6289d)